src/analyze.ts

```diff
@@ -15,6 +15,7 @@ import resolve from './resolve-dependency.js';
 //@ts-ignore
 import nodeGypBuild from 'node-gyp-build';
 import { Job } from './node-file-trace';
+import { fileURLToPath } from 'url';
```

Comment:
Maybe add `pathToFileURL` here to so both are imported on the same line instead of two separate lines?